src/components/features/MainMenuListItem.tsx

```diff
@@ -0,0 +1,48 @@
+import { Box, ListItemIcon, ListItemText, MenuItem } from '@mui/material'
+import { NavLink } from 'react-router-dom'
+
+interface MainMenuListItemProps {
+  title: string
+  Icon: React.ElementType
+  link: string
+}
+
+export default function MainMenuListItem({ title, Icon, link }: MainMenuListItemProps) {
+  return (
+    <MenuItem
+      sx={{
+        padding: 0,
+        '&:hover': {
+          backgroundColor: 'primary.100',
+          color: 'primary.main',
+          '& .MuiSvgIcon-root, & .MuiSvgIcon-fontSizeMedium, & .css-wg28wd-MuiSvgIcon-root & MuiTypography-root, & .MuiTypography-body1, & .MuiListItemText-primary, & .css-10hburv-MuiTypography-root':
```

Comment:
you can remove this big string and inside `Boz` on line `38` where you have the `color` just write `inherit` so the colors from MenuItem will be passed down using the `inherit` property and the icon will also get the color since it also has `inherit`

It should look something like this:

```
<MenuItem
      sx={{
        padding: 0,
        '&:hover': {
          backgroundColor: 'primary.100',
          color: 'primary.main'
        }
}}>
     <Box
           component={NavLink}{
           style={
               ....your other props,
              color: 'inherit'
           }
</MenuItem>
```